packages/opentelemetry/lib/tracing/tracerProvider.js

```diff
@@ -0,0 +1,150 @@
+/* eslint-disable no-unused-vars */
+
+import { getBatchSpanProcessor } from './spanProcessorFactory';
+import { buildExporter, AVAILABLE_EXPORTERS } from './exporterFactory';
+import { NodeTracerProvider } from '@opentelemetry/node';
+import { registerInstrumentations, InstrumentationOption } from '@opentelemetry/instrumentation';
+import { ServerInstrumentation } from './serverInstrumenation';
+import { diag, DiagConsoleLogger, DiagLogLevel } from '@opentelemetry/api';
+import { SpanProcessor } from '@opentelemetry/tracing';
+import { ExporterConfig } from '@opentelemetry/exporter-jaeger';
+import _ from 'lodash';
+import { forEachPromise } from '../utils';
+
+//Delegate
+class TracerProvider {
+  constructor () {
+    this.provider = new NodeTracerProvider();
+    this.exporters = {};
```

Comment:
this is a bit confusing that this is an object and the config property is an array. Could we also align property naming to show that difference?

Reply:
done